spec/services/payment_providers/adyen_service_spec.rb

```diff
@@ -140,5 +140,33 @@
         expect(payment_provider_service).to have_received(:preauthorise)
       end
     end
+
+    context 'when succeeded refund event' do
+      let(:refund_service) { instance_double(CreditNotes::Refunds::AdyenService) }
+
+      let(:event_json) do
+        JSON.parse(event_response_json)['notificationItems'].
```

Comment:
Layout/DotPosition: Place the . on the next line, together with the method name. (https://rubystyle.guide#consistent-multi-line-chains)